src/librustc/mir/interpret/error.rs

```diff
@@ -441,7 +454,8 @@ impl<'tcx, O: fmt::Debug> fmt::Debug for EvalErrorKind<'tcx, O> {
             }
             NoMirFor(ref func) => write!(f, "no mir for `{}`", func),
             FunctionPointerTyMismatch(sig, got) =>
-                write!(f, "tried to call a function with sig {} through a function pointer of type {}", sig, got),
+                write!(f, "tried to call a function with sig {} through a\
+                       function pointer of type {}", sig, got),
```

Comment:
This removes the space.
```
"a\
  b"
```
is the string `"ab"`. [See playground](https://play.rust-lang.org/?gist=4404bbbedc65bf5b8fcecb3d4de76f34&version=stable&mode=debug&edition=2015).

You should add a space in front of the backslash.